src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -136,6 +139,13 @@ private HostType GetHostType()
             var projectionBuffer = _containedLanguage.DataBuffer as IProjectionBuffer;
             if (projectionBuffer != null)
             {
+                // Projection buffers can have source buffers of multiple supported types. 
+                // First check to see if the subject buffer is the TypeScript content type.
+                if (_containedLanguage.SubjectBuffer.ContentType.IsOfType(TypeScript))
```

Comment:
What is this for? HostType is the type of document you're contained within, and shouldn't have anything to do with the content type of the subject buffer itself.